pkg/transforms/mqttsecret.go

```diff
@@ -176,9 +194,32 @@ func (sender *MQTTSecretSender) connectToBroker(ctx interfaces.AppFunctionContex
 	return nil
 }
 
+func (sender *MQTTSecretSender) setRetryData(ctx interfaces.AppFunctionContext, exportData []byte) {
+	if sender.persistOnError {
+		ctx.SetRetryData(exportData)
+	}
+}
+
+func (sender *MQTTSecretSender) onConnected(_ MQTT.Client) {
+	sender.lc.Tracef("MQTT Broker for export connected")
+}
+
+func (sender *MQTTSecretSender) onConnectionLost(_ MQTT.Client, _ error) {
+	sender.lc.Tracef("MQTT Broker for export lost connection")
+
```

Comment:
Newline

Reply:
gofmt should has removed... leaving for now.